fuels-contract/src/parameters.rs

```diff
@@ -8,6 +8,8 @@ pub struct TxParameters {
     pub gas_price: u64,
     pub gas_limit: u64,
     pub byte_price: u64,
+    // Use u32 as per the spec
+    pub maturity: u32,
```

Comment:
Comment isn't really needed, since the compiler will error if you use a different type (or, should!).
```suggestion
    pub maturity: u32,
```